validation/src/test/scala/hmda/validation/rules/lar/quality/Q001Spec.scala

```diff
@@ -0,0 +1,67 @@
+package hmda.validation.rules.lar.quality
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+import org.scalacheck.Gen
+
+class Q001Spec extends LarEditCheckSpec with BadValueUtils {
+
+  val config = ConfigFactory.load()
+  val loanAmount = config.getInt("hmda.validation.quality.Q001.loan.amount")
+  val multiplier = config.getInt("hmda.validation.quality.Q001.incomeMultiplier")
+
+  property("Valid with non-numeric income") {
+    forAll(larGen, Gen.alphaStr) { (lar, x) =>
+      val newApplicant = lar.applicant.copy(income = x)
+      val newLar = lar.copy(applicant = newApplicant)
+      newLar.mustPass
+    }
+  }
+
+  val irrelevantLoanAmount: Gen[Int] = Gen.choose(Int.MinValue, loanAmount - 1)
+
+  property(s"Valid whenever loan less than $loanAmount") {
+    forAll(larGen, irrelevantLoanAmount) { (lar, x) =>
+      val newLoan = lar.loan.copy(amount = x)
+      val newLar = lar.copy(loan = newLoan)
+      newLar.mustPass
+    }
+  }
+
+  val irrelevantIncome: Gen[Int] = Gen.choose(Int.MinValue, 0)
+
+  property("Valid whenever Income equal or less than 0") {
+    forAll(larGen, irrelevantIncome) { (lar, x) =>
+      val newApplicant = lar.applicant.copy(income = x.toString)
+      val newLar = lar.copy(applicant = newApplicant)
+      newLar.mustPass
+    }
+  }
+
+  val relevantIncome: Gen[Int] = Gen.choose(1000, 100000)
+  val validMultiplier: Gen[Int] = Gen.choose(1, multiplier - 1)
+
+  property(s"Valid when loan less than $multiplier times income") {
+    forAll(larGen, relevantIncome, validMultiplier) { (lar, x, m) =>
+      val newLoan = lar.loan.copy(amount = x * m)
```

Comment:
I'd like to see tests for the edge cases--the ones that make this edit barely pass or barely fail.

e.g. `loan = income * 4.99` (or `loan = (income * 5) - 1`)should pass, but `loan = income * 5` should fail